src/clj_headlights/bigtable_io.clj

```diff
@@ -97,18 +98,37 @@
           (map get-columns-data)
           (zipmap familiy-names))]))
 
-(s/defn bigtable-io-read-table-raw
+(s/defn read-table-raw-from-options
+  "Reads data from `table` using `BigtableOptions`
+  object passed in. Must use `BigtableOptions$Builder`
+  to construct options"
   [pipeline :- pcollections/PCollectionType
    name :- s/Str
-   project :- s/Str
-   {:keys [instance user-agent table]}]
+   table :- s/Str
+   bigtable-options :- BigtableOptions]
   (-> pipeline
       (.apply (str "read-from-bigtable-" name)
               (-> (BigtableIO/read)
-                  (.withBigtableOptions ^BigtableOptions$Builder (options-builder project instance user-agent))
+                  (.withBigtableOptions bigtable-options)
                   (.withTableId table)))))
 
+(s/defn bigtable-io-read-table-raw
```

Comment:
This would break compatibility, but how do you guys feel like removing bigtable-io from the prefix of these functions since it is already in the namespace so we don't have things like: `bigtable-io/bigtable-io-read-table-raw`?

Reply:
Yeah, the prefix should go away.

Reply:
Maybe we can remove the prefix and bind the old names for backward compatibility (while we deprecate the usage) until we have a major release.